src/main/java/tds/common/data/mapping/ResultSetMapperUtility.java

```diff
@@ -29,10 +32,36 @@ public static Instant mapTimeStampToInstant(ResultSet rs, String columnLabel) th
      * @param time The {@link java.time.Instant} to convert
      * @return A {@link java.sql.Timestamp} representation of the {@link java.time.Instant}
      */
-    public static Timestamp mapInstantToTimestamp(Instant time) {
+    public static Timestamp mapJodaInstantToTimestamp(Instant time) {
         return time == null
             ? null
             : Timestamp.valueOf(LocalDateTime.ofInstant(time, ZoneOffset.UTC));
     }
 
+    /**
+     * Map a {@link org.joda.time.Instant} to a {@link java.sql.Timestamp}
+     *
+     * @param time the {@link org.joda.time.Instant} to convert
+     * @return the converted instant
+     */
+    public static Timestamp mapInstantToTimestamp(org.joda.time.Instant time) {
```

Comment:
Isn't this one the joda time version, so shouldn't it be named `mapJodaInstantToTimestamp`